src/status_im/common/lightbox/style.cljs

```diff
@@ -1,9 +1,9 @@
-(ns status-im.contexts.chat.messenger.lightbox.style
+(ns status-im.common.lightbox.style
   (:require
     [quo.foundations.colors :as colors]
     [react-native.platform :as platform]
     [react-native.reanimated :as reanimated]
-    [status-im.contexts.chat.messenger.lightbox.constants :as c]))
+    [status-im.common.lightbox.constants :as c]))
```

Comment:
I see that sometimes this namespace is imported as `c`, and sometimes is imported as `constants`. We should unify the naming IMO.